release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -572,6 +572,11 @@ As of {product-title} 4.14, `DeploymentConfig` objects are deprecated. `Deployme
 
 Instead, use `Deployment` objects or another alternative to provide declarative updates for pods.
 
+[id="ocp-4-14-ztp-talm-defaultcatsrc-update"]
+=== Operator-specific CatalogSource CRs used in {gitops-shortname} ZTP are deprecated
+
+From {product-title} {product-version}, you must only use the `DefaultCatSrc.yaml` `CatalogSource` CR when updating Operators with {cgu-operator-first}. All other `CatalogSource` CRs are deprecated. For more information about `DefaultCatSrc` CR, see xref:../scalability_and_performance/ztp_far_edge/ztp-talm-updating-managed-policies.adoc#talo-operator-update_ztp-talm[Performing an Operator update].
```

Comment:
```suggestion
From {product-title} {product-version}, you must only use the `DefaultCatSrc.yaml` `CatalogSource` CR when updating Operators with {cgu-operator-first}. All other `CatalogSource` CRs are deprecated and are planned to removed in the <deprecation_timeline>. Red{nbsp}Hat will provide bug fixes and support for this feature during the current release lifecycle, but this feature will no longer receive enhancements and will be removed. For more information about `DefaultCatSrc` CR, see xref:../scalability_and_performance/ztp_far_edge/ztp-talm-updating-managed-policies.adoc#talo-operator-update_ztp-talm[Performing an Operator update].
```
Consider following more closely the recommended text from [the SSG's deprecation notice template](https://redhat-documentation.github.io/supplementary-style-guide/#release-notes-doc-texts) to ensure that users fully understand what "deprecation" means in this context. I've suggested one possible revision above.